crates/typst/src/layout/pages/finalize.rs

```diff
@@ -1,4 +1,4 @@
-use super::LayoutedPage;
+use super::LaidOutPage;
```

Comment:
This is a rename of an object, which would be an api break, if that's not a problem, it's easy enough to drop this (note that it's a different spelling and a different commit from the above variations)